### ISSUE TITLE
error... while installing the sample-data during upgrade

### ISSUE THREAD

**Issue by neophyt3** [magento/magento2#4172]
## Steps to reproduce
1. download Magento-CE-2.0.4-2016-03-31-10-48-19.zip
2. install magento 2 through web
3. copy the sample data app/code and pub/media to magento_install_dir/app and magento_install_dir/pub
4. then run bin/magento setup:upgrade
## Actual result 

Error:
  [ReflectionException]
  Class Magento\Catalog\Model\ResourceModel\Product\Gallery does not exist
##

**Comment by mazhalai** [comment]
@pradeepchauhan you need to run `composer install` after step 3.
BTW, there is an easier way than doing this, download the zip with sample data, or run `bin/magento sampledata:deploy` and then `bin/magento setup:upgrade`

**Comment by neophyt3** [comment]
@mazhalai , thanks! it worked!
After step 2 I ran bin/magento sampledata:deploy and then bin/magento setup:upgrade

**Comment by Laura021** [comment]
Using **sampledata:deploy** & **setup:upgrade** is not a real solution. 

I am using the sample data from ZIP  because I need to add my own custom sample data. 

I followed the instructions from [devdocs](http://devdocs.magento.com/guides/v2.0/install-gde/install/cli/install-cli-sample-data.html)  and during the  **setup:upgrade** I get the same error: 

`
  [ReflectionException]                                                     
  Class Magento\Catalog\Model\ResourceModel\Product\Gallery does not exist 
`

right before 

`Module 'Magento_CatalogSampleData':`

Any ideas what's going on ? 
Thanks!

**Comment by davidalger** [comment]
@Laura021 Please do not comment on closed issues. If you are encountering a bug, please open a new issue with a complete description and steps to reproduce following the format of the example. You can do this by clicking "New Issue" above on the right.
